include/boost/beast/core/detail/bind_default_executor.hpp

```diff
@@ -67,16 +67,26 @@ class bind_default_executor_wrapper
             h_, this->get());
     }
 
+    // The invocation hook is no longer defined because it customises behaviour
+    // without forwarding to a user's hook.
+
+#if 0
```

Comment:
Note that `bind_default_executor.hpp` is unused.... I don't remember writing it, or what it was for.